pgmpy/factors/continuous/LinearGaussianCPD.py

```diff
@@ -0,0 +1,128 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import division
+
+import numpy as np
+from scipy.stats import multivariate_normal
+
+from pgmpy.extern import six
+from pgmpy.factors import ContinuousFactor
+
+
+class LinearGaussianCPD(ContinuousFactor):
+    u"""
+    For, X -> Y the Linear Gaussian model assumes that the mean
+    of Y is a liner function of mean of X and the variance of Y does
+    not depend on X.
+
+    For example,
+    p(Y|X) = N(-2x + 0.9 ; 1)
+
+    Let Y be a continuous variable with continuous parents
+    X1 ............ Xk . We say that Y has a linear Gaussian CPD
+    if there are parameters β0,.........βk and σ2 such that,
+
+    p(Y |x1.......xk) = N(β0 + β1 + ......... + βk ; σ2)
```

Comment:
Shouldn't this be: `p(Y | x1, x2, ...., xk) = N(\beta_0 + x1 \beta_1 + ... + xk \beta_k ; \sigma_2)` ?